.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npm test
```

Comment:
This has likely been replaced by running the Husky init, as it replaces everything with the defaults, but we should be running `lint-staged` here.
```suggestion
npx lint-staged
```